crates/db_views_actor/src/community_follower_view.rs

```diff
@@ -11,19 +17,37 @@ use lemmy_db_schema::{
 
 type CommunityFollowerViewTuple = (Community, Person);
 
+sql_function!(fn coalesce(x: diesel::sql_types::Nullable<diesel::sql_types::Text>, y: diesel::sql_types::Text) -> diesel::sql_types::Text);
```

Comment:
This should go in utils.rs (mod functions) and be called `coalesce_text()` to avoid conflict with other coalesce functions (Im adding one for time in my PR). Would be great if this could be made generic to work for multiple types, or better yet built into diesel.